cmd/multi.go

```diff
@@ -155,17 +163,18 @@ func doMultiScheduling(args []string, workDir, forcedQueue, queuesToAvoid string
 
 	unique := scheduler.UniqueString()
 	outputRoot := filepath.Join(workDir, unique)
+	now := time.Now().Format("20060102150405")
```

Comment:
20060102-150405
Would be a lot more readable, but presumably will need more changes here and in wrstat to work with the new format. I think it's worth it...